measures/ApplyUpgrade/resources/constants.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Constants
-  NumApplyUpgradeOptions = 25
+  NumApplyUpgradeOptions = 100
```

Comment:
This requires an update to RTD at docs/read_the_docs/advanced_tutorial/increasing_upgrade_options.rst.

Reply:
Addressed: [91cdc87](https://github.com/NREL/resstock/pull/1329/commits/91cdc876c3c5b6ee45337aae5eb294062f4c138a)